orbit/pkg/table/extension.go

```diff
@@ -5,6 +5,10 @@ import (
 	"time"
 
 	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/macadmins/osquery-extension/tables/chromeuserprofiles"
+	"github.com/macadmins/osquery-extension/tables/fileline"
+	"github.com/macadmins/osquery-extension/tables/puppet"
```

Comment:
Double checking that `macadmins/...` contains tables that are not mac only, AFAICS `extension.go` will compile for all platforms.

Reply:
I've just checked, it's g2g. I got confused that `github.com/macadmins/osquery-extension` only had extension tables for macOS.